indico/modules/rb_new/client/js/common/bookings/BookingDetailsModal.jsx

```diff
@@ -197,18 +361,22 @@ class BookingDetailsModal extends React.Component {
         const dates = {startDate: startDt, endDate: endDt};
         const times = {startTime: moment(startDt).format('HH:mm'), endTime: moment(endDt).format('HH:mm')};
         const recurrence = this.getRecurrence(repetition);
-        const legendLabels = [
-            {label: Translate.string('Occurrence'), color: 'orange'},
-        ];
+        const legendLabels = [{label: Translate.string('Occurrence'), color: 'orange'}];
         const showAccept = canAccept && !isAccepted;
         const showActionButtons = (!isCancelled && !isRejected && (canCancel || canReject || showAccept));
         return (
             <Modal open onClose={this.handleCloseModal} size="large" closeIcon>
                 <Modal.Header styleName="booking-modal-header">
-                    <Translate>Booking Details</Translate>
+                    <span styleName="header-text">
+                        <Translate>Booking Details</Translate>
+                    </span>
+                    <span styleName="booking-status">
+                        {/* eslint-disable-next-line react/destructuring-assignment */}
+                        {this.renderBookingStatus(this.props.booking)}
```

Comment:
Why pass a prop to a method of the same component? You could just do this inside `renderBookingStatus`:

    const {booking: {isPending, isAccepted, isCancelled, isRejected, rejectionReason}} = this.props;

Reply:
Oh, I think my brain just froze at the point of writing this.